packages/mobile/src/transactions/TransferFeedItem.tsx

```diff
@@ -169,7 +170,10 @@ export class TransferFeedItem extends React.PureComponent<Props> {
       comment = null
     } else if (type === TransactionTypes.FAUCET) {
       contactImage = <Image source={faucetIcon} style={styles.image} />
-      fullName = 'Celo'
+      const testnetName = DEFAULT_TESTNET
```

Comment:
let's use lodash for this, would be neater: https://stackoverflow.com/questions/38084396/lodash-title-case-uppercase-first-letter-of-every-word

Reply:
Ah right there is lodash.